notebooks/tm/building_footprints/4_extract_features.ipynb

```diff
@@ -0,0 +1,851 @@
+{
```

Comment:
<p>For consistency, revise to:</p><p>    "ADM4_PCODE",</p><p>&nbsp;&nbsp;&nbsp;&nbsp;"brgy_area",</p><p>    "bldg_count",</p><p>&nbsp;&nbsp;&nbsp;&nbsp;"bldg_total_area",</p><p>&nbsp;&nbsp;&nbsp;&nbsp;"bldg_mean_area",</p><p>&nbsp;&nbsp;&nbsp;&nbsp;"bldg_count_lt100sqm",</p><p>&nbsp;&nbsp;&nbsp;&nbsp;"bldg_count_100_200sqm",</p><p>&nbsp;&nbsp;&nbsp;&nbsp;"bldg_count_gt200sqm",</p><p>&nbsp;&nbsp;&nbsp;&nbsp;"bldg_area_density",</p><p>&nbsp;&nbsp;&nbsp;&nbsp;"bldg_pct_brgy_area"</p><p> </p><p>Kindly apply this also in the column parameter in the <code>vzs.create_zonal_stats</code> function</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/thinkingmachines/lacuna-fund-climate-health/pull/11/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>

Reply:
<p>Noting that will rename columns according to recent ERD. </p>